### ISSUE TITLE
segmentation fault when import tensorboardX

### ISSUE THREAD

**Issue by Liu0329** [lanpa/tensorboardX#178]
When I import tensorboardX
then shows
Segmentation fault (core dumped)
I have tried conda, pip, pip from source.

**Comment by lanpa** [comment]
Did you also import other packages?

**Comment by Liu0329** [comment]
No.

**Comment by rxqy** [comment]
Hi, I met a similar problem here, but my problem is with my tensorflow
Can you successfully import tensorflow in python?

**Comment by lanpa** [comment]
@Liu0329 @rxqy, I can't reproduce without further information. I recommend using anaconda's virtual environment to build do the test.

**Comment by rxqy** [comment]
Nope, I fixed the problem by reinstalling tensorflow, there was something wrong with it.

**Comment by lanpa** [comment]
@rxqy what's the error message?

**Comment by rxqy** [comment]
When I import tensorboard,  I got segmentation fault,
then I tried to import tensorflow, there was another strange error, but I didn't really remember.
So I just reinstalled tensorflow, and the problem is fixed, guess something is wrong in my environment.

**Comment by lanpa** [comment]
import tensorboard or import tensorboardX? Did you install from source or from pip?

**Comment by rxqy** [comment]
tensorboardX, from conda

**Comment by lanpa** [comment]
I haven't tried conda yet. Would you try with source or pip?

**Comment by rxqy** [comment]
ehhhhh, I fixed the problem myself. Everything's fine with me.
Saw a similar issue, and thought maybe I could help.

**Comment by Liu0329** [comment]
@rxqy @lanpa I use python 3.6 with pytorch 0.4. The error shows when I enter python and import nothing but tensorboardX. It is not related to other modules.

Python 3.6.5 |Anaconda, Inc.| (default, Mar 29 2018, 18:21:58)
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorboardX
Segmentation fault (core dumped)

**Comment by lanpa** [comment]
@Liu0329 Please use https://stackoverflow.com/questions/16731115/how-to-debug-a-python-segmentation-fault?rq=1 to get more information

**Comment by MilanKolkata** [comment]
@Liu0329 I also have the same problem and have no idea to fix it. Have you fixed it for now?

**Comment by Liu0329** [comment]
@MilanKolkata No

**Comment by lanpa** [comment]
@MilanKolkata Please use https://stackoverflow.com/questions/16731115/how-to-debug-a-python-segmentation-fault?rq=1 to get more information (the result of pip list will be helpful too)

**Comment by szrlee** [comment]
same issue

**Comment by szrlee** [comment]
> same issue

after update this package, the problem was solved.
```
libprotobuf: 3.6.0-hd28b015_0 conda-forge --> 3.6.0-hdbcaa40_0
```

**Comment by dragon9001** [comment]
@szrlee your solution works for me, I update my package using:
conda update libprotobuf

the version change is:
    libprotobuf: 3.6.1-hd28b015_0 conda-forge --> 3.6.1-hd408876_0 defaults

**Comment by leixiang90** [comment]
conda install libprotobuf=3.6.1
this works for me

**Comment by lamnguyen95** [comment]
I was facing the same issue when installing tensorboardX offline.

Turns out that was version mismatch between protobuff and libprotobuff. Both must be at version 3.6.1

**Comment by lanpa** [comment]
addressed in fix #467.

**Comment by RizhaoCai** [comment]
> conda install libprotobuf=3.6.1

Thank you. It also works for me.
